inc/cron_pull.inc

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * @file
+ * Pull cron function for Loco Translate module.
+ */
+
+/**
+ * Pull assets & translations from Loco.
+ *
+ * @param string $langcode
+ *   The langcode to pull from.
+ * @param array $settings
+ *   The configurations needed to pull from Loco.
+ */
+function loco_translate_cron_pull($langcode, array $settings) {
+  $loco_pull = \Drupal::service('loco_translate.loco_api.pull');
+  $translations_import = \Drupal::service('loco_translate.translations.import');
+  $file_system = \Drupal::service('file_system');
+
+  $loco_pull->setApiClientFromConfig();
+  $status = $settings['status'] ?? NULL;
+  $response = $loco_pull->fromLocoToDrupal($langcode, $status);
```

Comment:
To avoid confusion I propose to remove to rename this method `->pull()`. LocoPull do nothing in Drupal.

Reply:
`pull` is not enougt. We don't understand if we pull into Drupal or if we pull into Loco.

I agree the name is too much ... but pull is not explicit

Reply:
what about fetch ?